src/components/expanded-state/chart/ChartExpandedStateHeader.js

```diff
@@ -93,7 +95,7 @@ export default function ChartExpandedStateHeader({
   return (
     <Container testID={'expanded-state-header'} showChart={showChart}>
       <Stack space={'20px'}>
-        {shouldUseRainbowCoinEffect && asset?.iconUrl ? (
+        {(isRainbowToken || shouldUseRainbowCoinEffect) && asset?.iconUrl ? (
```

Comment:
is the plan for `shouldUseRainbowCoinEffect` to remain false or do you want it as a global control still? (e.g, should this || be &&)

Reply:
Left the old flag in as a dev flag that shows it for all tokens